miniwin/miniwin/src/include/miniwin_d3drmframe_p.h

```diff
@@ -28,6 +28,8 @@ struct Direct3DRMFrameImpl : public Direct3DRMObjectBase<IDirect3DRMFrame2> {
 	HRESULT SetMaterialMode(D3DRMMATERIALMODE mode) override;
 	HRESULT GetChildren(IDirect3DRMFrameArray** children) override;
 
+	D3DCOLOR m_backgroundColor = 0xFFFFFFFF;
```

Comment:
Is there a reason you set the default background color of a frame to white, and that of a viewport to black?

When I tried some simple d3drm examples, they had black backgrounds.

Reply:
Oh, that's my bad i wanted to set it as black actually. I guess that just shows it never actually rendered an uninitialized scene.

Reply:
```suggestion
	D3DCOLOR m_backgroundColor = 0xFF000000;
```